### ISSUE TITLE
Install raw image disk [FR]

### ISSUE THREAD

**Issue by bignaux** [ps2homebrew/hdl-dump#49]
### Checks

- [x] I have checked existing feature requests for duplicates and found none

### Describe the FR

Provide a way to install raw image as easy as game. Need refactoring code to separate ISO handling from APA partition creation.
I thought [libblkid](http://ftp.ntu.edu.tw/linux/utils/util-linux/v2.26/libblkid-docs/index.html)  could to the job to get info about the block device going to be injected.

### Describe the solution you'd like

`hdl_dump inject <device> <disc_file> <options>`
inject disc_file according to its proprieties (iso, ext2, swap, fat32, pfs ... ) 

### Additional context

Now i've device-mapping i could use host mkfs and mount it to populate, so i'm concentrate on mkpart-like.

**Comment by bignaux** [comment]
For reference : https://github.com/util-linux/util-linux/issues/1614
Could interest @frno7

**Comment by frno7** [comment]
Indeed, with issue https://github.com/frno7/linux/issues/39, thanks @bignaux!

**Comment by bignaux** [comment]
**Guideline for PS2 Linux**

Use UUIDs for boot and swap partitions to share them between different linux. See https://systemd.io/DISCOVERABLE_PARTITIONS/ .If i understand,  it has to be at APA level to mimic GPT UUID, so it's right place (linux installer hdl_dump) to speak about it.

* boot : ?
* Swap : 0657fd6d-a4ab-43c4-84e5-0933c84b4f4f
* Home Partition 933ac7e1-2eb4-4f13-b844-0e14e2aef915

we could also use others.

**need**

* add : uchar uuid[16] somewhere in APA header, like mbr field.
* modify https://gitlab.com/ps2max/linux/linux/-/blob/ps2-v3.8/block/partitions/ps2.c to add the partition UUID to the partition info structure of the kernel  and remove code relying on PS2_LINUX_ID. https://github.com/frno7/linux/blob/ps2-main/block/partitions/msdos.c#L99-L108 is a perfect example for that. Make it compatible with HDL partition scheme, to be able to put loader in PPAA and be able to be launchable in OSD/OPL and other stuff (that would be symetrical to my userland libblkid implementation). Note this driver doesn't need to be devel on PS2 so devel could start on latest and be cherry pick later by frno7.
* adapt OPL-launcher to be able to launch kernel (with kernel parameters) from some system pfs as it did for OPL.
